src/components/TextPopup/TextPopup.js

```diff
@@ -61,9 +76,17 @@ const TextPopup = () => {
       data-element="textPopup"
       ref={popupRef}
       style={{ ...position }}
-      onClick={() => dispatch(actions.closeElement('textPopup'))}
+      onClick={onClose}
     >
       <CustomizablePopup dataElement="textPopup">
+        {fullAPI && isTextEditEnabled && (
```

Comment:
It looks like `isTextEditEnabled` is always false so there's no way to turn it on. As I mentioned in another comment(or during screenshare?) we don't need to have the extra state in the store to handle toggling. The `ActionButton` component renders a `Button` component, which handles `instance.disableElements`.